src/ngx_http_lua_kong_module.c

```diff
@@ -670,5 +683,223 @@ ngx_http_lua_kong_get_upstream_ssl_verify(ngx_http_request_t *r,
 
     return ctx->upstream_ssl_verify;
 }
+# endif
+
+int
+ngx_http_lua_kong_ffi_var_load_index(u_char *name_data,
```

Comment:
change to a directive and a single FFI function to get all indexes from known variables.